src/components/Todo/HistoryTodo.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { FC, useState } from "react";
+
+import { useNotification } from "src/hooks";
+import { useDispatch } from "react-redux";
+import { changeStatus, deleteTodo } from "src/store/todo/actions";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors, getShadow, typography } from "src/styles";
+import { i18n } from "src/locale";
+
+import { ITodo } from "src/types";
+
+import { Button } from "../Button";
+
+interface Props {
+  todo: ITodo;
+}
+
+export const HistoryTodo: FC<Props> = ({ todo }) => {
```

Comment:
I think it also should be one component (Todo) with diffrent buttons depends on the` todo.status` property